seqr/views/utils/json_to_orm_utils.py

```diff
@@ -68,7 +68,8 @@ def update_model_from_json(model_obj, json, user, allow_unknown_keys=False, immu
             raise ValueError('Cannot edit field {}'.format(orm_key))
         if allow_unknown_keys and not hasattr(model_obj, orm_key):
             continue
-        if getattr(model_obj, orm_key) != value:
+        model_value = getattr(model_obj, orm_key)
+        if (model_value or value) and model_value != value:
```

Comment:
It means no update from `Null` to `[]` or '{}` and vice versa. Is that what we want?

Reply:
yes

Reply:
oh but I just realized we do care about `None` vs `False` for nullable booleans. I'll change this back